pytests/common_test_server.py

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
This file appears unchanged from the tests/ folder.  Can you symlink one to the other instead of duplicating it?

Reply:
Since PyPi support for Python2 is effectively becoming deprecated, it would be best if this were the original now and the one in the test/ directory were the symlink